lib/ro_crate_ruby.rb

```diff
@@ -19,6 +19,9 @@
 require_relative './ro_crate/model/organization'
 require_relative './ro_crate/reader'
 require_relative './ro_crate/writer'
-#
+
+module ROCrateRuby
```

Comment:
Is this needed?

Reply:
oh sorry, I thought it was (looking at another gem as a model) but actually its not.